src/practices/JavaScript/DependenciesVersionMajorLevel.ts

```diff
@@ -4,71 +4,76 @@ import { Package } from '../../inspectors/IPackageInspector';
 import { PackageInspectorBase, SemverLevel } from '../../inspectors/package/PackageInspectorBase';
 import { PracticeEvaluationResult, PracticeImpact, ProgrammingLanguage } from '../../model';
 import { DxPractice } from '../DxPracticeDecorator';
-import { IPractice } from '../IPractice';
+import { PracticeBase } from '../PracticeBase';
+import { ReportDetailType } from '../../reporters/ReporterData';
 
 @DxPractice({
   id: 'JavaScript.DependenciesVersionMajorLevel',
   name: 'Update Dependencies of Major Level',
   impact: PracticeImpact.small,
-  suggestion: 'Keep the dependencies updated to have all possible features. Use, for example, Renovate Bot.',
+  suggestion: 'Keep the dependencies updated to have all possible features. Use, for example, npm-check-updates.',
```

Comment:
But it's product placement. We don't want to prefer any paid service. Github has e.g. Dependabot and it's free only for open-sources and it is better than Renovate bot because you can customize its behaviour..